app/src/main/java/com/eveningoutpost/dexdrip/FollowerManagementActivity.java

```diff
@@ -12,25 +12,27 @@
 import android.widget.ListView;
 import android.widget.Toast;
 
-import com.eveningoutpost.dexdrip.ShareModels.FollowerManager;
 import com.eveningoutpost.dexdrip.ShareModels.Models.ExistingFollower;
+import com.eveningoutpost.dexdrip.ShareModels.Models.InvitationPayload;
+import com.eveningoutpost.dexdrip.ShareModels.ShareRest;
 import com.eveningoutpost.dexdrip.utils.ActivityWithMenu;
 
 import java.util.List;
 
-import rx.functions.Action1;
+import retrofit.Callback;
+import retrofit.Response;
+import retrofit.Retrofit;
 
 /**
  * Created by stephenblack on 8/11/15.
  */
 public class FollowerManagementActivity extends ActivityWithMenu {
-    static String FollowerManagementActivity = Home.class.getName();
     public static String menu_name = "Follower Management";
-
+    static String TAG = Home.class.getName();
     ListView existingFollowersView;
     Button addFollowerButton;
-    FollowerManager followerManager;
-    Action1<List<ExistingFollower>> existingFollowerListener;
+    ShareRest shareRest;
+    Callback<List<ExistingFollower>> existingFollowerListener;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/matthias-granberry/xDrip-Experimental/file/1296698328/issues/source?bid=2735053&fileBranchId=2947762#l35)